server/src/main/java/com/sauron/model/BankRequest.java

```diff
@@ -0,0 +1,36 @@
+package com.sauron.model;
+
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+
+public class BankRequest<R> {
+
+    private final HttpMethod httpMethod;
+    private final String path;
+    private final HttpEntity<?> entity;
+    private final ParameterizedTypeReference<R> responseType;
+
+    public BankRequest(HttpMethod httpMethod, String path, HttpEntity<?> entity, ParameterizedTypeReference<R> responseType) {
```

Comment:
Probably it should be somehow related to Bank entity (do we need both anyway?)

Reply:
I think it would be necessary to add bankId to BankRequest entity.